examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>Ideally only the required modules should be loaded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done!